tensorflow/compiler/xla/service/gpu/tests/fusion_logical_index_test.cc

```diff
@@ -53,8 +53,8 @@ ENTRY entry {
 ; CHECK:  %[[index_3:.*]] = urem i32 %[[base_1]], 400
 ; CHECK:  %[[base_3:.*]] = udiv i32 %[[base_1]], 400
 ; CHECK:  %[[index_2:.*]] = urem i32 %[[base_3]], 480
-; CHECK:  %[[pointer:.*]] = getelementptr inbounds [1 x [64 x [480 x [400 x float]]]], [1 x [64 x [480 x [400 x float]]]]* %5, i32 0, i32 0, i32 %[[index_1]], i32 %[[index_2]], i32 %[[index_3]]
-; CHECK:  store float %[[result_value:.*]], float* %[[pointer]], align 4
+; CHECK:  %[[pointer:.*]] = getelementptr inbounds [1 x [64 x [480 x [400 x float]]]], ptr %2, i32 0, i32 0, i32 %[[index_1]], i32 %[[index_2]], i32 %[[index_3]]
```

Comment:
FYI.  I have an upstream PR with this fix:

https://github.com/tensorflow/tensorflow/pull/56479